package.json

```diff
@@ -65,7 +65,7 @@
   "dependencies": {
     "babel-plugin-debug-macros": "^0.3.3",
     "broccoli-funnel": "^3.0.3",
-    "ember-cli-babel": "^7.21.0",
+    "ember-cli-babel": "7.18",
```

Comment:
This seems suspect.  We should probably remove it.

Reply:
Yup, it was leftover from testing, as linking was unhappy with babel issues at some point